staging/src/k8s.io/apiserver/pkg/endpoints/handlers/proxy.go

```diff
@@ -57,16 +57,7 @@ func (r *ProxyHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 
 	var httpCode int
 	var requestInfo *request.RequestInfo
-	defer func() {
```

Comment:
this is only the special proxy handler. Something similar must be in the other endpoints.